### PR TITLE
Add SS58 registry

### PR DESCRIPTION
**Pull request by joepetrowski**
A manual draft of what the autogenerated table should look like.

### CODE REVIEW

**Comment by lsaether** [comment]
Can we use the registry to generate the `<option>` tags in the address changer? 

https://github.com/w3f/polkadot-wiki/blame/master/docs/learn-accounts.md#L355-368

**Comment by Swader** [comment]
Excellent idea

File: docs/build-ss58-registry.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+id: build-ss58-registry
+title: SS58 Registry
+sidebar_label: SS58 Registry
+---
+
+Substrate-based chains, including the Polkadot and Kusama relay chains, use an [SS58 encoding](https://github.com/paritytech/substrate/wiki/External-Address-Format-(SS58)) for their address formats. This page serves as a registry for teams to see which chain corresponds to a given prefix, and which prefixes are availabe.
```
Comment (krichard410) [inline docs/build-ss58-registry.md:7]:
```suggestion
Substrate-based chains, including the Polkadot and Kusama relay chains, use an [SS58 encoding](https://github.com/paritytech/substrate/wiki/External-Address-Format-(SS58)) for their address formats. This page serves as a registry for teams to see which chain corresponds to a given prefix, and which prefixes are available.
```
spelling mistake 🤣

Comment (joepetrowski) [reply docs/build-ss58-registry.md:7]: you caught me :)

**Comment by Swader** [comment]
This is now even easier with https://github.com/polkadot-js/common/tree/master/packages/networks#readme so we should use this.

**Comment by laboon** [comment]
What is the status of this?

**Comment by Swader** [comment]
No change, didn't have time

**Comment by laboon** [comment]
Superseded by #1581

**Pull request closed without merging**